### PR TITLE
Fixes #10383: If I click on "Regenerate all policies", it clears the cache of changes, and slows down a lot the web interface

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/10383

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
You need to rebase your pull request

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/ClearCache.scala [hunk 1]
```diff
@@ -101,9 +102,55 @@ class ClearCache extends DispatchSnippet with Loggable {
...
+                    , details = EventLog.emptyDetails
+                    , reason = Some("Clearing cache for: node configuration, recent changes, compliance and node info at user request")
+                  )
+                )
+             )
+    } yield {
+      set
+    }) match {
+      case eb:EmptyBox =>
+        val e = eb ?~! "Error when logging the cache event"
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/ClearCache.scala]: The error may not be wile logging the cache event but also when clearNodeConfigurationCache fails

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/ClearCache.scala [hunk 2]
```diff
@@ -72,27 +72,28 @@ class ClearCache extends DispatchSnippet with Loggable {
...
+          val modId = ModificationId(uuidGen.newUuid)
+          eventLogRepository.saveEventLog(
+              modId
+            , ClearCacheEventLog(
                 EventLogDetails(
                     modificationId = Some(modId)
                   , principal = CurrentUser.getActor
                   , details = EventLog.emptyDetails
-                  , reason = None))) match {
+                  , reason = Some("Node configuration caches deleted on user request")
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/ClearCache.scala]: "cache" instead of "caches"

**Comment by fanf** [comment]
PR rebased

**Pull request merged**